File: Python/1042 - Sort Simples.py
a, b, c = map(int,input().split(" "))

vetor = [a, b, c]

vetor_ordenado = sorted(vetor)

for i in range(3):
    print(vetor_ordenado[i])

print()

for i in range(3):
    print(vetor[i])

File: Clojure/1042 - Sort Simples.clj
(let [entrada (map #(Integer/parseInt %) (clojure.string/split (read-line) #"\s+"))
      [a b c] entrada
      vetor [a b c]
      vetor-ordenado (vec (sort vetor))]
  (doseq [x vetor-ordenado]
    (println x))
  (println)
  (doseq [x vetor]
    (println x)))